### ISSUE TITLE
Disable runner_description cardinality

### ISSUE THREAD

**Issue by sluongng** [mvisonneau/gitlab-ci-pipelines-exporter#205]
It would be nice to be able to disable runner_description inside `processJobMetrics`.

The high cardinality is not ideal for use cases where metric storage is limited while having a lot of runners.

**Comment by mvisonneau** [comment]
absolutely!

**Comment by mvisonneau** [comment]
I pushed a change to make this configurable : https://github.com/mvisonneau/gitlab-ci-pipelines-exporter/commit/4bd60cba173cbb1a00717d880c633a9cce7a7472

You can leverage this new feature by using the following configuration:

```yaml
projects_default:
  pull:
    pipeline:
      jobs:
        runner_description:
          # Export the description of the runner which ran the job
          # (optional, default: true)
          enabled: true

          # Whenever the description of a runner will match this regexp
          # The regexp will be used as the value of the runner description instead
          # (optional, default: "shared-runners-manager-(\d*)\.gitlab\.com")
          aggregation_regexp: shared-runners-manager-(\d*)\.gitlab\.com
```

**Comment by mvisonneau** [comment]
this has been shipped as part of [v0.4.8](https://github.com/mvisonneau/gitlab-ci-pipelines-exporter/releases/tag/v0.4.8), closing this issue.
